packages/clerk-js/webpack.config.js

```diff
@@ -250,6 +271,8 @@ const prodConfig = ({ mode }) => {
     output: {
       filename: '[name].js',
       libraryTarget: 'commonjs',
+      chunkFormat: 'commonjs',
```

Comment:
This was added due to the inclusion of Webpack's [auto public path runtime module](https://github.com/webpack/webpack/blob/dae16ad11e2d2accddc549a92082da3d2e667b01/lib/runtime/AutoPublicPathRuntimeModule.js#L74), which was unable to calculate a public path in our Chrome extension tests. Since this only impacts the CommonJS build, switching the chunk format to `commonjs` causes Webpack to emit chunks in CJS rather than in a format requiring a URL (I believe it defaulted to `importScripts` which is used for Web Workers).